pyproject.toml

```diff
@@ -17,37 +17,33 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-pycountry = "^20.7.3"
+python = "^3.8"
```

Comment:
Why can't we continue to support python 3.7 for the time being?

Reply:
We can, but I suppose I'd be good to have a wider conversation on how many versions of Python we want to support and for how long.